src/tasks/netSdk/NetSdkTaskHelper.ts

```diff
@@ -0,0 +1,138 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { IActionContext } from "@microsoft/vscode-azext-utils";
+import * as os from 'os';
+import { WorkspaceFolder, l10n } from "vscode";
+import { vsDbgInstallBasePath } from "../../debugging/netcore/VsDbgHelper";
+import { ext } from "../../extensionVariables";
+import { RunContainerBindMount, Shell, composeArgs, withArg, withNamedArg } from "../../runtimes/docker";
+import { getValidImageName } from "../../utils/getValidImageName";
+import { getDockerOSType } from "../../utils/osUtils";
+import { quickPickProjectFileItem } from "../../utils/quickPickFile";
+import { quickPickWorkspaceFolder } from "../../utils/quickPickWorkspaceFolder";
+import { DockerContainerVolume } from "../DockerRunTaskDefinitionBase";
+import { defaultVsCodeLabels } from "../TaskDefinitionBase";
+import { DockerTaskExecutionContext, addVolumeWithoutConflicts, getDefaultContainerName, getDefaultImageName, getMounts } from "../TaskHelper";
+import { NetCoreTaskHelper } from "../netcore/NetCoreTaskHelper";
+
+/**
+ * Native architecture of the current machine in the RID format
+ * {@link https://github.com/dotnet/runtime/blob/main/src/libraries/Microsoft.NETCore.Platforms/src/runtime.json}
+ */
+export type RidCpuArchitecture =
+    | 'x64'
+    | 'x86'
+    | 'arm64'
+    | 'arm'
+    | 'ppc64le'
+    | 'mips64'
+    | 's390x'
+    | string;
+
+export const NET_SDK_RUN_TASK_SYMBOL = 'dotnet-container-sdk';
```

Comment:
Would prefer PascalCase for these constants

Reply:
sounds good!